Toolset/home.livecodescript

```diff
@@ -723,12 +672,359 @@ private command revInternal__SetupSplash
+   put "false" into tPropsArray["cREVMsgShowUIMessages"]
+   put "false" into tPropsArray["cSEHidePalettes"]
+   put "120" into tPropsArray["cREVImageWidth"]
+   put "500" into tPropsArray["tooltip delay"]
+   put "" into tPropsArray["cUserOrganization"]
+   put "417,72" into tPropsArray["cREVMsgTabRectSingle Line"]
+   put "false" into tPropsArray["cREVMsgShowUIBackScripts"]
+   put "539,432" into tPropsArray["cREVGeoTopLeft"]
+   put "70378" into tPropsArray["cREVDBCode"]
+   put "" into tPropsArray["cColumnVisibility"]
+   put "276" into tPropsArray["cREVScriptHeight"]
+   put "2" into tPropsArray["cGridSize"]
+   put "483,214,1015,476" into tPropsArray["cVariableWatcherRect"]
+   put "false" into tPropsArray["cShowrevProps"]
+   put "Pixels" into tPropsArray["cREVRulerMeasurement"]
+   put "GreenYellow" into tPropsArray["GreenYellow"]
+   put "100" into tPropsArray["cMacOSScrollbarWidth"]
+   put "1,2,3,4,5,7" into tPropsArray["cControlTabs"]
+   put "" into tPropsArray["cREVCustomWidth"]
+   put "500" into tPropsArray["cREVToolTipDelay"]
+   put "250" into tPropsArray["cTabButtonwidth"]
+   put "true" into tPropsArray["cGoAlign"]
+   put "1:80" into tPropsArray["http proxy"]
+   put "2.6.1" into tPropsArray["cVersionNo"]
+   put "0" into tPropsArray["cREVRotatePrev2"]
+   put "993,712" into tPropsArray["cREVPropertyPaletteTopLeft"]
+   put "true" into tPropsArray["cLiveColor"]
+   put "modification date" into tPropsArray["cRecentSort"]
+   put "true" into tPropsArray["cDocsFirstRun"]
+   put "86400" into tPropsArray["cUpdate,MaxAutoPopupFrequency"]
+   put "true" into tPropsArray["cREVExtension"]
+   put "102" into tPropsArray["cMenuWidth"]
+   put "417,72" into tPropsArray["cREVMsgTabRectcard id 1147"]
+   put "468,131" into tPropsArray["cAppearanceTopLeft"]
+   put "false" into tPropsArray["cREVDefaultMenubar"]
+   put "" into tPropsArray["cREVMultipleLineHistory"]
+   put "417,113" into tPropsArray["cREVMsgTabRectbackScripts"]
+   put "true" into tPropsArray["cGrid"]
+   put "30" into tPropsArray["cNumRecent"]
+   put "false" into tPropsArray["cShowAsHandlers"]
+   put "false" into tPropsArray["revDocsAlreadySeen"]
+   put "120" into tPropsArray["cREVImageHeight"]
+   put "82" into tPropsArray["cButtonWidth"]
+   put "true" into tPropsArray["cREVProfileReadOnlyWarning"]
+   put "" into tPropsArray["cHTTPProxy"]
+   put "6,7,9" into tPropsArray["cCardTabs"]
+   put "" into tPropsArray["cREVScriptFont"]
+   put "16" into tPropsArray["cMacOSScrollbarHeight"]
+   put "true" into tPropsArray["cFirstRun"]
+   put "14" into tPropsArray["cMacOSProgressScrollbarHeight"]
+   put "" into tPropsArray["cREVTemplateMasterName"]
+   put "mouseMove" into tPropsArray["suppressedMessages"]
+   put "true" into tPropsArray["cAutoFormat"]
+   put "true" into tPropsArray["cREVScriptShortcut"]
+   put "false" into tPropsArray["cShowRevolutionStacks"]
+   put "22" into tPropsArray["cMenuHeight"]
+   put "1067,251,1882,827" into tPropsArray["cRevDocsRect"]
```

Comment:
again each stack should have a default width and height and gravity and set it when the stack is opened to ensure that it isn't being pushed to the wrong screen